scripts/cluster_network.py

```diff
@@ -170,6 +170,45 @@ def weighting_for_country(n, x):
     return (w * (100. / w.max())).clip(lower=1.).astype(int)
 
 
+def get_feature_for_hac(n, buses_i, feature=None): #buses_i = n.buses.index
```

Comment:
```suggestion
def get_feature_for_hac(n, buses_i=None, feature=None):

    if buses_i is None:
        buses_i = n.buses.index
```

Would it make sense to make `buses_i` an optional argument?